coderd/database/modelmethods.go

```diff
@@ -4,6 +4,17 @@ import (
 	"github.com/coder/coder/coderd/rbac"
 )
 
+func (s ApiKeyScope) ToRBAC() rbac.Scope {
```

Comment:
We should add an alias for `api_key_scope` in `sqlc.yaml` so this name becomes `APIKeyScope`.